module-api/src/main/java/kitchenpos/tablegroup/application/TableGroupService.java

```diff
@@ -17,32 +18,29 @@
 @Service
 @Transactional(readOnly = true)
 public class TableGroupService {
-    private final OrderService orderService;
+    private final OrderRepository orderRepository;
     private final TableService tableService;
     private final TableGroupRepository tableGroupRepository;
 
-    public TableGroupService(final OrderService orderService,
+    public TableGroupService(final OrderRepository orderRepository,
         final TableService tableService,
         final TableGroupRepository tableGroupRepository) {
-        this.orderService = orderService;
+        this.orderRepository = orderRepository;
         this.tableService = tableService;
         this.tableGroupRepository = tableGroupRepository;
     }
 
     @Transactional
     public TableGroup create(final TableGroupRequest tableGroupRequest) {
         tableGroupRequest.validateOrderTableSize();
-        List<OrderTable> orderTables = tableService.findAllByIdIn(tableGroupRequest.getOrderTableIds());
-        return tableGroupRepository.save(new TableGroup(new OrderTables(orderTables)));
+        OrderTables orderTables = tableService.findAllByIdIn(tableGroupRequest.getOrderTableIds());
+        return tableGroupRepository.save(new TableGroup(orderTables));
     }
 
 
     @Transactional
     public void ungroup(final Long tableGroupId) {
         final TableGroup tableGroup = findTableGroupById(tableGroupId);
-        final List<OrderTable> orderTables = tableService.findAllByTableGroupId(tableGroupId);
-
-        orderService.validateOrderStatusNotIn(orderTables, Arrays.asList(OrderStatus.COOKING, OrderStatus.MEAL));
 
         tableGroup.getOrderTables().ungroup();
```

Comment:
`get()`을 하기보다는 메시지를 보내보면 좋겠네요 :)

```suggestion
        tableGroup.ungroup();
```